### PR TITLE
#478 Handle XXX / FIXME

### PR DESCRIPTION
**Pull request by snimavat**
connect #478

Some of the FIXMEs for #478

### CODE REVIEW

File: rally-domain/src/integration-test/groovy/yakworks/rally/orgs/OrgRepoTests.groovy [hunk 1]
```diff
@@ -340,11 +340,15 @@ class OrgRepoTests extends Specification implements DomainIntTest {
     }
 
     void "test find org by sourceid"() {
```
Comment (basejump) [inline rally-domain/src/integration-test/groovy/yakworks/rally/orgs/OrgRepoTests.groovy:342]: name "test lookup by sourceid", add lookup to other test name to so it more obvious

File: examples/testify/src/integration-test/groovy/gorm/tools/async/ParallelToolsSpecSpec.groovy [hunk 2]
```diff
@@ -17,28 +17,19 @@ class ParallelToolsSpecSpec extends Specification {
...
-        // Project.withSession {
-        //     Project.count() == 50
-        // }
         //starting org count
         Org.count() == 100
-
         list.size() == 50
 
         when:
         // FIXME #339 how is this working, transaction thats set on the test should not be rolling into parallelTools?
```
Comment (basejump) [inline examples/testify/src/integration-test/groovy/gorm/tools/async/ParallelToolsSpecSpec.groovy:25]: lets deal with FIXME here while in here add comment on this question

File: gorm-tools/src/test/groovy/gorm/tools/validation/DomainValidationSpec.groovy [hunk 3]
```diff
@@ -15,7 +15,7 @@ class DomainValidationSpec extends Specification implements GormHibernateTest  {
     static List entityClasses = [ValidationEntity]
 
     Closure doWithGormBeans() { { ->
-        //FIXME this should be in the commonGormBeans, configed so it picks up more
+        //FIXME Already in commonGormBeans, but here it needs 'searchClasspath = true'
```
Comment (basejump) [inline gorm-tools/src/test/groovy/gorm/tools/validation/DomainValidationSpec.groovy:18]: can remove the FIXME from this since its clear now why.

**Review by basejump — CHANGES_REQUESTED** [review]

File: examples/testify/src/integration-test/groovy/gorm/tools/async/ParallelToolsSpecSpec.groovy (hunk 2 above)
Comment (snimavat) [reply examples/testify/src/integration-test/groovy/gorm/tools/async/ParallelToolsSpecSpec.groovy:25]: I havent been able to figureout it yet, so i did not remove fixme @basejump

**Pull request merged**